app/assets/javascripts/discourse/controllers/preferences/emails.js.es6

```diff
@@ -51,6 +56,25 @@ export default Ember.Controller.extend(PreferencesTabController, {
     { name: I18n.t("user.email_digests.every_two_weeks"), value: 20160 }
   ],
 
+  @computed("model.user_option.email_messages_level")
+  emailMessageLevelAway(level) {
+    return level === 1;
+  },
+
+  @computed("model.user_option.email_level")
+  emailLevelAway(level) {
+    return level === 1;
+  },
```

Comment:
This should be a `const` in javascript rather than `1`.

Also it's a good candidate for `Ember.computed.equal`. I'd suggest replacing both computed properties with that macro.